qubes/storage/reflink.py

```diff
@@ -32,13 +32,24 @@
 import os
 import subprocess
 import tempfile
+import platform
 from contextlib import contextmanager, suppress
 
 import qubes.storage
 import qubes.utils
 
-FICLONE = 1074041865        # defined in <linux/fs.h>, assuming sizeof(int)==4
-LOOP_SET_CAPACITY = 0x4C07  # defined in <linux/loop.h>
+HOST_MACHINE = platform.machine()
+
+if HOST_MACHINE == "x86_64":
+    FICLONE = 1074041865        # defined in <linux/fs.h>, assuming sizeof(int)==4
+    LOOP_SET_CAPACITY = 0x4C07  # defined in <linux/loop.h>
+elif HOST_MACHINE == "ppc64le":
+    FICLONE = -2147183607
```

Comment:
That is essentially what I ran, but I used `%d` instead of `%lu`.

`%lu` returns `2147783689` which is equivalent to `-2147183607` in 2's compliment form.

Reply:
Ah, okay.